bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/builder/AbstractUserTaskBuilder.java

```diff
@@ -155,6 +148,24 @@ public B zeebeFollowUpDateExpression(final String expression) {
     return zeebeFollowUpDate(asZeebeExpression(expression));
   }
 
+  @Override
+  public B zeebeFormId(final String formId) {
+    final ZeebeFormDefinition formDefinition =
+        getCreateSingleExtensionElement(ZeebeFormDefinition.class);
+    formDefinition.setFormId(formId);
+    return myself;
+  }
+
+  /**
+   * Marks the user task as native Zeebe user task.
+   *
+   * @return the builder object
+   */
+  public B zeebeUserTask() {
```

Comment:
I actually didn't add it to the interface because that breaks backward compatibility regarding Revapi.

Reply:
But in hindsight, your argumentation is also correct, I would say.

Reply:
Regarding Revapi, I don't think this is a breaking change, so it would have been fine to add an exclusion.

However, I'm fine with leaving it as it is.